test/test_nn.py

```diff
@@ -2155,7 +2155,9 @@ def forward(self, X):
                 # Cayley map
                 # If X is skew-symmetric it returns an orthogonal matrix
                 Id = torch.eye(X.size(0), device=X.device)
-                return torch.linalg.solve(Id + X, Id - X)
+                # We call contiguous because solve returns a tensor with strides that are Fortran-contiguous
```

Comment:
Do you have a code sample that reproduces this outside of reparametrization?
Is it because you have `t.weight` that is contiguous and has a `t.weight.grad` that is also contiguous.
But then you do `t.weight.set_(other)` and that one is not while `t.weight.grad` is still contiguous?